charts/argo-cd/templates/argocd-server/certificate.yaml

```diff
@@ -30,4 +30,10 @@ spec:
   privateKey:
     {{- toYaml . | nindent 4 }}
   {{- end }}
+  {{- if .Values.server.certificate.usages }}
+  usages:
+    {{- range .Values.server.certificate.usages }}
+    - {{ . | quote }}
+    {{- end }}
+  {{- end }}
```

Comment:
Maybe this is simpler? 🙋 

```yaml
  {{- with .Values.server.certificate.usages }}
  usages:
    {{- toYaml . | nindent 4 }}
  {{- end }}
```

Reply:
Probably, I wasn't sure actually, my first time writing helm.
I did see example in the code for the `range`

Reply:
FYI
[here](https://github.com/argoproj/argo-helm/pull/2143/files#diff-6395eca2b798a2c7a98337b4f336f77d333f193264e733ff78a412faf08f4807R29-R32) is the existing sample.

Reply:
Thanks :)
Updated, hope it's fine